apps/site/assets/ts/stop/components/DisplayTime.tsx

```diff
@@ -121,6 +121,25 @@ function DelayedTimeDetails(): JSX.Element {
   );
 }
 
+const CancelledTimeDetails = (): JSX.Element => {
+  const { time, targetDate } = useContext(DepartureContext);
+
+  return (
+    <>
```

Comment:
**nitpick:** you don't technically need this `React.Fragment`; it's typically needed to wrap 2+ elements